packages/manager/modules/preloader/src/constants.js

```diff
@@ -0,0 +1,26 @@
+export const WELCOME_MESSAGES = [
+  ['Welcome to the ', 'OVHcloud Control Panel'],
```

Comment:
how does this works ?

Reply:
Documentation would be nice, it doesn't look like anything we've done yet

Reply:
Yes, in fact we don't use angular for that and we don't want to import a lot of js libraries for a preloader (I don't want to add a preloader of the preloader :D).
But we have to split welcome messages in 2 parts for display purposes. I could update to add an object or something else if you want.

Reply:
yes but how do you map the sentence to the user language ?

Reply:
We display messages in all languages with a CSS animation to fadeIn/fadeOut.